t/ffi_platypus_type_enum.t

```diff
@@ -167,11 +167,6 @@ subtest 'define errors' => sub {
     match qr/rev must be either 'int', or 'str'/,
   );
 
-  is(
```

Comment:
Instead of removing this test, please substitute another unsupported type, like a scalar ref.